faiss_vector_posting.go

```diff
@@ -288,20 +298,40 @@ func (i *vectorIndexWrapper) Size() uint64 {
 // InterpretVectorIndex returns a construct of closures (vectorIndexWrapper)
 // that will allow the caller to -
 // (1) search within an attached vector index
-// (2) close attached vector index
-// (3) get the size of the attached vector index
-func (sb *SegmentBase) InterpretVectorIndex(field string, except *roaring.Bitmap) (
+// (2) search limited to a subset of documents within an attached vector index
+// (3) close attached vector index
+// (4) get the size of the attached vector index
+func (sb *SegmentBase) InterpretVectorIndex(field string, filtered bool,
```

Comment:
If we're going to need this variable, let's keep the naming consistent (even across repositories) - `filtered` or `requiresFiltering` (whichever makes most sense).

Reply:
Rename to `requiresFiltering` per your `blevesearch/scorch_segment_api` PR.